test/t/test_apt_mark.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+
+
+@pytest.mark.bashcomp(cmd="apt-mark")
+class TestAptMark:
+    @pytest.mark.complete("apt-mark ")
+    def test_1(self, completion):
+        assert all(x in completion for x in "auto manual remove showinstall showremove \
+        hold minimize-manual showauto showmanual unhold install \
```

Comment:
Not sure if it's just the patch viewer, or if we have a bit of an indentation issue here.

Anyway no need to use backslashes for line continuation, just use multiple string constants, they'll auto concatenate like
```python
... for x in (
    "auto manual remove showinstall showremove "
    "hold minimize-manual showauto showmanual unhold install "
    # [...]
)